Assets/Scripts/KatamariMovement.cs

```diff
@@ -60,69 +60,13 @@ void Start()
     // Update is called once per frame
     void Update()
     {
+
         horizontalInput = Input.GetAxis("Horizontal");
         verticalInput = Input.GetAxis("Vertical");
         hitInput = Input.GetAxis("Jump");
         stopInput = Input.GetAxis("Stop");
 
-
-
-
-        // turn the power bar on and off depending if we can hit hit the ball
-        if (power_bar_active)
-        {
-            power_bar.SetActive(true);
-            isGolfHitMode = true;
-            this.GetComponent<LineRenderer>().enabled = true;
-
-
-        }
-        else
-        {
-            power_bar.SetActive(false);
-            this.GetComponent<LineRenderer>().enabled = false;
-        }
-
-        if (resourceManager.can_hit())
-        {
-            power_bar_active = true;
-        }
-
-
-        // SLOW MOTION
-
-        float flyMovementDeadTime = 0.2f;
-        bool isFlyMovementDeadTime = resourceManager.GetTimeSinceLastHit() >= flyMovementDeadTime;
-
-        if (power_bar_active == false && hitInput > 0.5 && isFlyMovementDeadTime)
-        {
-            time_manager.SlowMotion();
-        }
-
-
-        else
-        {
-            time_manager.NormalTime();
-
-        }
-
-
-
-
-        if (power_bar_active == false && hitInput > 0.5 && !isFlyMovementDeadTime)
-        {
-            hitInput = 0;
-        }
-
-
-        // makes the power bar go up and down and apply strength if in golf mode
-        PowerBar();
-
-        strikeStrength = power * 2500;
-
-
-
-        if (hitInput >= 0.5)
+        if (hitInput >= 0.5 && power_bar_active == true)
```

Comment:
`== true` is sort of redundant here
```suggestion
        if (hitInput >= 0.5 && power_bar_active)
```